website/content/en/docs/cloud-providers/AWS/launch-templates.md

```diff
@@ -0,0 +1,236 @@
+
+Finally, **the majority of Launch Template fields should not be set** (or will have no effect), such as network interfaces and instance type. 
+
+## Important Fields
+
+When creating a custom launch template, the AMI and User Data are the defining characteristics. 
+
+### AMI
+
+AMI (Amazon Machine Image), is the base image/VM for a launch template.
+
+[Review the instructions for importing a VM to AWS.](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) Note the AMI id generated by this process, such as, 
+`ami-074cce78125f09d61`.
+
+### User Data - Autoconfigure
+
+Importantly, the AMI must support automatically connecting to a cluster based
+on "user data", or a base64 encoded string passed to the instance at startup.
+The syntax and purpose of the user data varies between images. The Karpenter
+default OS, Amazon Linux 2 (AL2), accepts shell scripts (bash commands). 
+
+[AWS calls data passed to an instance at launch time "user
+data".](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-shell-scripts)
+
+In the default configuration, Karpenter uses an EKS optimized version of AL2 and passes the hostname of the Kubernetes API server, and a certificate. The EKS Optimized AMI includes a `bootstrap.sh` script which connects the instance to the cluster, based on the passed data. 
+
+Alternatively, you may reference AWS's [`bootstrap.sh`
+file](https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh)
+when building a custom base image. 
+
+```
+#!/bin/bash
+/etc/eks/bootstrap.sh <my-cluster-name> \
+--kubelet-extra-args <'--max-pods=40'> \
+--b64-cluster-ca <certificateAuthority> \
+--apiserver-endpoint <endpoint> 
+--dns-cluster-ip <serviceIpv4Cidr>
+--use-max-pods false
+```
+
+Note, you must populate this command with live values. Karpenter will
+not change the user data in the launch template. 
+
+Encode using yaml function `!Base64` yaml function or `cat userdata.sh | base64 > userdata-encoded.txt` shell command. 
+
+**Bootstrap Script Parameters**
+
+The sample bootstrap script requires information to join the cluster.
+
+These values may be found using:
+```
+aws eks describe-cluster --name MyKarpenterCluster 
+```
+
+**Kubelet Arguments**
+
+Specifying max-pods can break Karpenter's binpacking logic (it has no way to know what this setting is). If karpenter attempts to pack more than this number of pods, the instance may be oversized, and additional pods will reschedule.
+
+## Situational Fields 
+
+Configure these values in response to a particular use case, such as nodes interacting with another AWS service, or using EBS storage on the node. 
+
+### Instance Profile - IAM
+
+The launch template must include an "instance profile" -- a set of IAM roles. 
+
+The instance profile must include all the permissions of the default Karpenter
+node instance profile. For example, permission to run containers and manage

```

Comment:
I'm not sure what you mean by `run containers`? The Instance profile shouldn't have anything special to "run containers", unless I'm misunderstanding.

Reply:
I'm going to replace this with a link to "AmazonEKSWorkerNodePolicy"